src/lib/constants/dev-environment.ts

```diff
@@ -28,11 +28,26 @@ export const DEV_ENVIRONMENT_WORDPRESS_CACHE_KEY = 'wordpress-versions.json';
 
 export const DEV_ENVIRONMENT_WORDPRESS_VERSION_TTL = 86400; // once per day
 
-export const DEV_ENVIRONMENT_PHP_VERSIONS: Record< string, string | undefined > = {
-	'8.0': 'ghcr.io/automattic/vip-container-images/php-fpm:8.0',
-	8.2: 'ghcr.io/automattic/vip-container-images/php-fpm:8.2',
-	8.1: 'ghcr.io/automattic/vip-container-images/php-fpm:8.1',
-	7.4: 'ghcr.io/automattic/vip-container-images/php-fpm:7.4',
+interface PhpImage {
+	image: string;
+	label: string;
+}
+
+export const DEV_ENVIRONMENT_PHP_VERSIONS: Record< string, PhpImage > = {
+	'8.0': {
+		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.0',
+		label: '8.0 (recommended)',
+	},
+	8.2: {
+		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.2',
+		label: '8.2 (experimental)',
+	},
+	8.1: { image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.1', label: '8.1' },
+	7.4: { image: 'ghcr.io/automattic/vip-container-images/php-fpm:7.4', label: '7.4 (outdated)' },
+	8.3: {
+		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.3',
+		label: '8.3 (experimental, not officially supported)',
+	},
```

Comment:
We should make 8.1 the recommended version and flag 8.0 as 

```suggestion
	8.1: { image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.1', label: '8.1 (recommended)' },
	8.2: {
		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.2',
		label: '8.2 (experimental)',
	},
	8.3: {
		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.3',
		label: '8.3 (experimental, not supported)',
	},
	'8.0': {
		image: 'ghcr.io/automattic/vip-container-images/php-fpm:8.0',
		label: '8.0 (EOL soon)',
	},
	7.4: { image: 'ghcr.io/automattic/vip-container-images/php-fpm:7.4', label: '7.4 (EOL; not supported)' },
```